crates/solvers/src/util/conv.rs

```diff
@@ -45,6 +45,22 @@ pub fn u256_to_biguint(i: &U256) -> BigUint {
     BigUint::from_bytes_be(&bytes)
 }
 
+pub fn u256_to_bigdecimal(i: &U256) -> BigDecimal {
+    BigDecimal::new(u256_to_biguint(i).into(), 0)
+}
+
+pub fn bigint_to_u256(i: &BigInt) -> Option<U256> {
+    if i.sign() != num::bigint::Sign::Plus {
+        return None;
+    }
+    biguint_to_u256(i.magnitude())
```

Comment:
This returns `None` if `i` is zero since its sign is `NoSign` according to the [docs](https://docs.rs/num/latest/num/struct.BigInt.html#method.sign), I don't think it's intended.